test/TestProjects/Models-Cadl/Generated/BaseModelWithDiscriminator.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace ModelsInCadl
+{
+    /// <summary> Base model with discriminator property. </summary>
+    public partial class BaseModelWithDiscriminator
+    {
+        /// <summary> Initializes a new instance of BaseModelWithDiscriminator. </summary>
+        public BaseModelWithDiscriminator()
+        {
```

Comment:
Is this the final output or just a placeholder for the test project?

Reply:
This isn't a final output cause discriminator isn't handled in autorest.csharp yet